serverless-sample/app/aws/Makefile

```diff
@@ -0,0 +1,10 @@
+
+build:	
+	CGO_ENABLED=0 GOOS=linux go build -v -installsuffix nocgo -o .builds/binary-aws main.go
+upload:	
+	docker build -t $(ECR_REPO):$(tag) .
+	docker push $(ECR_REPO):$(tag)
+clean:
+	rm -rf .builds
+	docker rmi $(ECR_REPO):$(tag)
+	AWS_PAGER="" aws ecr batch-delete-image --repository-name $$(cut -d'/' -f2 <<< $(ECR_REPO)) --image-ids imageTag=$(tag)
```

Comment:
Do we actually want to delete the image from the cloud container registry on clean? it could mean that a previous build with the latest tag gets deleted. If the person who ran that, never runs upload the image will be gone...